### ISSUE TITLE
overlay when streaming in OBS

### ISSUE THREAD

**Issue by soporteimatt22** [Hotrian/OVRdrop-Public#79]
Hello today I bought the OVRdrop Is there a way that the window that I can see in the VR is also reflected in the PC, because when transmitting with twich the viewers do not see the window when I turn my hand to see the chat that I

attached illustrative image

[![obs.jpg](https://i.postimg.cc/85JgyfFY/obs.jpg)](https://postimg.cc/cgSzCHB7)

**Comment by Hotrian** [comment]
Yes, but you'll need to open and capture the VR Mirror in Steam instead of the game itself - I think the OBS SteamVR Plugin does this automatically. I think SteamVR calls this the Desktop View now.
